src/main/java/org/threadly/litesockets/tcp/ssl/SSLClient.java

```diff
@@ -246,37 +254,32 @@ public boolean isEncrypted() {
       if(ssle.getHandshakeStatus() == NEED_WRAP) {
         writeForce(ByteBuffer.allocate(0));
       }
-      if(this.ce != null) {
-        ce.getThreadScheduler().schedule(new Runnable() {
+      if(this.seb != null) {
```

Comment:
This could be something that could use the new "Watchdog" I am (slowly) working on.